analyzers/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicSyntaxHelper.cs

```diff
@@ -182,6 +182,7 @@ private static bool IsOn(this ExpressionSyntax expression, SyntaxKind onKind)
                 PropertyStatementSyntax x => x.Identifier,
                 SimpleArgumentSyntax x => x.NameColonEquals?.Name.Identifier,
                 SimpleNameSyntax x => x.Identifier,
+                QualifiedNameSyntax x => x.Right.Identifier,
```

Comment:
The `GetIdentifier` methods between CSharp and VB are out of sync (The cs version returns SimpleNameSyntax while VB returns SyntaxToken). This was caused by me in one of the record struct PRs (It was mentioned in the todo list of the PR. We ran out of time and so I put it on my todo list to fix that). Can you try to refactor the cs version of `GetName` and `GetIdentifier` to match the VB version? I think the VB version is doing it the right way.

Reply:
It's a rabbit hole, so I'd like to avoid it in this PR. But in general, the idea makes sense as the C# method is called `GetIdentifier`, but returns something else.

Reply:
@martin-strecker-sonarsource if it's a  necessary change in the codebase, could you please open an issue on the repo for the required change (to describe the problem, how it got out of sync etc)?

Like this the team will have visibility.